tests/integration/security/util/connection/checker.go

```diff
@@ -52,6 +52,14 @@ func (c *Checker) Check() error {
 				return err
 			}
 		}
+		if c.ExpectMTLS {
+			err := results.CheckMTLSForHTTP()
+			gotMtls := err == nil
+			if gotMtls != c.ExpectMTLS {
+				return fmt.Errorf("%s to %s:%s using %s: expected mtls=%v, got mtls=%v",
```

Comment:
you don't have to specify from and to right? We have already create different test case with these info as sub test name.

Reply:
I am just copying what is nearby, but I agree